addons/website/static/src/snippets/s_alert/options.js

```diff
@@ -0,0 +1,27 @@
+odoo.define('website.s_alert_options', function (require) {
+'use strict';
+
+var options = require('web_editor.snippets.options');
+
+options.registry.Alert = options.Class.extend({
+    //--------------------------------------------------------------------------
+    // Options
+    //--------------------------------------------------------------------------
+
+    /**
+     * @override
+     */
+    selectType: function (previewMode, widgetValue, params) {
+        // compatibility: we need `alert-delta` to be removed, which will be the
+        // case by selecting another color with the colorpicker.
```

Comment:
Yep but since the colorpicker was not removing alert-xxx, I thought it was normal behavior, so it was important to remove alert-delta. This is not relevant anymore indeed.